geode-management/src/main/java/org/apache/geode/management/runtime/RebalanceInfo.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.management.runtime;
+
+import org.apache.geode.management.api.JsonSerializable;
+
+public class RebalanceInfo implements JsonSerializable {
```

Comment:
this class could be renamed to be more general, like StringInfo

Reply:
I suspect `rebalance` will return a more complex result than a simple String, but it was stubbed out this way since this PR is focused on proving out the framework, not implementing the actual rebalance command yet.